gradle/dependenciesCheck.gradle

```diff
@@ -21,8 +21,20 @@ buildscript {
         }
     }
     dependencies {
-        classpath 'org.owasp:dependency-check-gradle:3.0.1'
+        classpath 'org.owasp:dependency-check-gradle:3.0.2'
     }
 }
 
-apply plugin: org.owasp.dependencycheck.gradle.DependencyCheckPlugin
+configure(projectsIncludedInDistribution()) {
+    apply plugin: org.owasp.dependencycheck.gradle.DependencyCheckPlugin
+
+    dependencyCheck {
+        failBuildOnCVSS = 8 // 10 is the maximum
+        skipConfigurations = ['jmh']
+    }
+}
+
+def projectsIncludedInDistribution() {
+    def excluded = [':buildScanPerformance', ':distributions', ':docs', ':integTest', ':internalIntegTesting', ':internalPerformanceTesting', ':internalTesting', ':performance', ':smokeTest', ':soak']
```

Comment:
⭕️ Cast to `Set` to make the `contains()` further down efficient.

Reply:
done